README.md

```diff
@@ -103,11 +103,11 @@ MIT
 
 ## TODO
 - [x] Move Ramda to peer dep.
-- [ ] Throw if `UA` is not passed in to `parse` as `ua-parser-js` fallsback to `window.navigator.userAgent` in a browser env.
+- [x] Throw if `UA` is not passed in to `parse` as `ua-parser-js` fallsback to `window.navigator.userAgent` in a browser env.
```

Comment:
NOTE: Didn't actually throw as it's a nicer API to accept undefined, e.g. `isBrowserName('Safari', req.headers['user-agent'])` can be performed without having to check whether the 'user-agent' header exists.